6-space-game/2-drawing-to-canvas/README.md

```diff
@@ -81,14 +81,14 @@ img.onload = () => {
 It's recommended to wrap the above in a construct like so, so it's easier to use and you only try to manipulate it when it's fully loaded:
 
 ```javascript
-async function loadAsset(path) {
```

Comment:
please readd async, otherwise you are good to go. Reason is the ussage in the solution:

```
window.onload = async() => {
  canvas = document.getElementById("canvas");
  ctx = canvas.getContext("2d");
  const heroImg = await loadTexture('assets/player.png')
  const enemyImg = await loadTexture('assets/enemyShip.png')

  ctx.fillStyle = 'black';
  ctx.fillRect(0,0, canvas.width, canvas.height);
  ctx.drawImage(heroImg, canvas.width/2 - 45, canvas.height - (canvas.height /4));
  createEnemies(ctx, canvas, enemyImg);
};
```

Reply:
> You can see that the await keyword is being used to wait for the textures to fully load and then we do the drawing

Of course, 'await' is good for waiting for async or promisified function (and yes await can be called from only async function).
But what I removed is `async` from an async function (loadAsset)  that returns a promise.
 The `loadAsset` would work well with async, however, I think that duplication of async and returning promise may need a strong educational purpose or forcing of the coding convention (since it's "duplication" :-) ).

In practice, before I remove `async`, I had checked codes in this repo to catch some coding convention or style.
I think more relevant codes from the solution would be `function loadTexture`

https://github.com/microsoft/Web-Dev-For-Beginners/blob/2258e0cdbaf279b1a88ac8a9e17621512de87b18/6-space-game/2-drawing-to-canvas/solution/app.js#L1-L9

This is quite an equivalent function with the `loadAsset` without async.

so the removal of `async` from `loadAsset` does
1. match with coding style with the solution
2. Perfectly work in the algorithms
3. remove unnecessary duplication (if it's not really needed).

Add. Oh, It could be that you thought an `async` from `async function run()` but not from `async function loadAsset`. If then, my answer would be simpler :-)

Reply:
I see your point, I agree